music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -432,6 +432,7 @@ async def cmd_volume_set(self, player_id: str, volume_level: int) -> None:
     async def cmd_stop(self, player_id: str) -> None:
         """Send STOP command to given player."""
         player = self.mass.players.get(player_id, raise_unavailable=False)
+        player.elapsed_time = time.time()
```

Comment:
this doesnt make any sense

Reply:
Well then there is a bug somewhere else.
With this it works as it should.
The problem is that when pausing, it goes back to the first song in the queue.
It also clears the list of songs already played.
Do you have any idea what it could be ?